Exercise_A/about.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<header>
+    <Div>
+        <a href="mailto: alex92335126@gmail.com"
```

Comment:
Hey Alex
Great work with this, when i clicked on this link i was able to send you an email